blocks/library/image/block.js

```diff
@@ -220,10 +224,69 @@ class ImageBlock extends Component {
 								onChange={ this.updateImageURL }
 							/>
 						) }
+						<div className="blocks-image-dimensions">
```

Comment:
I feel that these classNames don't follow our guidelines, should it be `blocks-image__dimentions`?